packages/datadog-plugin-fastify/test/index.spec.js

```diff
@@ -542,7 +542,198 @@ describe('Plugin', () => {
             })
           }
         })
+
+        describe('with tracer config codeOriginForSpans.enabled: true', () => {
```

Comment:
Everything LGTM at this point, but I'm wondering if this test could also be split to its own file. If it's too complicated we can keep it here for now but if it's easy I'd say let's do it.

Reply:
Not a problem. I've pushed a new commit with the tests split into two files